code/src/OSFramework/Helper/JSONParser.ts

```diff
@@ -63,56 +21,4 @@ namespace OSFramework.Helper {
             .toISOString()
             .substr(0, 10);
     }
```

Comment:
These 2 methods will stay here?

Reply:
They are being used by the ValidationMark. Does it make sense?